.buildkite/pipeline.yaml

```diff
@@ -113,10 +113,10 @@ steps:
       - seek-oss/aws-sm#v2.3.2:
           json-to-env:
             - json-key: .
-              secret-id: sdlc/prod/buildkite/devex_gcp_creds_base64
```

Comment:
Are these actually different creds or did the secret just get moved?

Reply:
Secrets got moved